rtv/curses_helpers.py

```diff
@@ -183,9 +185,77 @@ def init(cls):
         # Assign the terminal's default (background) color to code -1
         curses.use_default_colors()
 
+        # Set default colors
+        cls._colors.update( config.default_colors )
+        # Override default colors by theme colors
+        cls._colors.update( cls.load_colors(config.theme) )
+
         for index, (attr, code) in enumerate(cls._colors.items(), start=1):
             curses.init_pair(index, code[0], code[1])
-            setattr(cls, attr, curses.color_pair(index))
+            color = curses.color_pair(index)
+            # add format to color
+            for f in code[2:4]: color |= f
+            setattr(cls, attr, color)
+
+    @classmethod
+    def load_colors(cls, theme):
+        # get all available themes
+        themes = cls.load_themes()
+        # get theme to use if exists
+        theme  = themes[theme] if theme in themes.keys() else {}
+
+        # compute colors hash
+        colors = {}
+        # loop in theme entries
+        for key in theme:
+            values = []
+            for value in theme[ key ].strip().split(','):
```

Comment:
`strip()` doesn't remove spaces, only newlines and tabs.

`for value in theme[ key ].strip().replace(' ', '').split(','):`